people.json

```diff
@@ -27237,9 +27237,8 @@
       "wechat": "",
       "website": "",
       "youtube": "https://www.youtube.com/channel/UCedj-RrSa0d7h-oqWkHu8aQ",
-      "priority": 100,
       "category": [
-         "Staff"
+         "Emeritus"
```

Comment:
"Emeritus" isn't a valid value for now so it's causing the JSON Validate step to fail. Do you want to just remove yourself entirely for now? Until the day you remerge as Executive Director or Golden Kubestronaut?? ;)

Reply:
BAHAHAHA 🤣 makes sense to me, true…true…I'll have to come back one of these days as "Kubestronaut the Golden." Thank you, @cjyabraham! I'll make those changes shortly.